lib/linter.js

```diff
@@ -74,12 +74,19 @@ module.exports = function (cwd, args, text) {
 
     return JSON.stringify(fileConfig, null, '  ');
   }
+  if (currentOptions.fixToStdout && !stdin) {
```

Comment:
Should I check for `text` here? tbh, the check for text and the fallback to using files below is confusing. It seems `stdin` should *always* use stdin, even if the stdin is empty.